.github/workflows/stripe-patrons-data.yml

```diff
@@ -57,6 +56,50 @@ jobs:
             ~/.coursier
           key: sbt
 
-      - name: Build and upload to RiffRaff
+      - name: Build
         run: |
-          sbt "project stripe-patrons-data" riffRaffUpload
+          sbt "project stripe-patrons-data" assembly
+
+      - name: Upload to RiffRaff
```

Comment:
Migrate to using `actions-riff-raff` action for riffraff because it enables us to set the artifact name based on the build number